README.md

```diff
@@ -1,8 +1,8 @@
 <h5 align="center">
   <br>
-  <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
+  <a href="https://www.asyncapi.org"><img src="https://user-images.githubusercontent.com/60163079/157887904-c6328b3b-1423-4fd1-85ff-8d2bafe408d8.png" alt="AsyncAPI logo"></a>
```

Comment:
Can we have this image hosted somewhere else so that we don't depend on `user-images.githubusercontent.com`? Is this image available on the `brand` repo for example?
@ritik307 and @mcturco wdyt?

Reply:
Agreed! We could probably add an `assets` directory within this repo, add the image in there and then reference the path. But up to you guys how that gets organized in the files 👍

Reply:
Yeah, we should create new folder in the `.github` dir -> `.github/assets`